lib/elliptic/utils.js

```diff
@@ -147,4 +148,54 @@ function getJSF(k1, k2) {
 
   return jsf;
 }
-utils.getJSF = getJSF;
+exports.getJSF = getJSF;
+
+function lazyComputed(obj, name, computer) {
+  var key = '_' + name;
+  obj.prototype[name] = function lazyComputed() {
+    return this[key] !== undefined ? this[key] :
+           this[key] = computer.apply(this, arguments);
+  };
+}
+exports.lazyComputed = lazyComputed;
+
+function parseBytes(bytes) {
+  return typeof bytes === 'string' ? utils.toArray(bytes, 'hex') :
+                                     bytes;
+}
+exports.parseBytes = parseBytes;
+
+function intFromLE(bytes) {
+  return new bn(bytes, 'hex', 'le');
+}
+exports.intFromLE = intFromLE;
+
+function intToLE(num, padTo) {
+  var bytes = num.toArray('le');
+  while (bytes.length < padTo)
+    bytes.push(0);
+  return bytes;
+}
+exports.intToLE = intToLE;
+
+function pointToLEYoddX(point, length) {
```

Comment:
Are you sure you want these here? It might be nice to have encode methods on Point classes at some point? My feeling about utils is, if you export them, somebody might gonna use em.